tests_extractor.d

```diff
@@ -143,12 +201,17 @@ void main(string[] args)
     string inputDir;
     string outputDir = "./out";
     string ignoredFilesStr;
-    string modulePrefix = "";
+    string modulePrefix;
+    string attributesStr;
+    VisitorConfig visitorConfig;
 
     auto helpInfo = getopt(args, config.required,
             "inputdir|i", "Folder to start the recursive search for unittest blocks (can be a single file)", &inputDir,
             "outputdir|o", "Folder to which the extracted test files should be saved (stdout for a single file)", &outputDir,
-            "ignore", "Comma-separated list of files to exclude (partial matching is supported)", &ignoredFilesStr);
+            "ignore", "Comma-separated list of files to exclude (partial matching is supported)", &ignoredFilesStr,
+            "attributes|a", "Comma-separated list of UDAs that the unittest should have", &attributesStr,
+            "betterC", "Add custom extern(C) main method for running D's unittests", &visitorConfig.betterCOutput,
```

Comment:
Maybe it's better to rename this to `--main` as it's now more analogous to the `-main` switch of DMD?